### PR TITLE
adding docstrings to apigw.py

### PR DESCRIPTION
**Pull request by joshuaroot**
- adding docstring examples to apigw module

### CODE REVIEW

File: c7n/resources/apigw.py [hunk 1]
```diff
@@ -250,6 +302,28 @@ def process_task_set(self, client, task_set):
...
+            resource: rest-resource
+            filters:
+              - type: rest-method
+                key: authorizationType
+                value: NONE
+                op: eq
+            actions:
+              - type: update-method
+                patch:
+                  -
```
Comment (kapilt) [inline c7n/resources/apigw.py]: formatting looks off

**Comment by kapilt** [comment]
i assume these are tested?, or lifted from the extant unit tests?

File: c7n/resources/apigw.py [hunk 2]
```diff
@@ -68,6 +68,25 @@ def get_resources(self, resource_ids):
...
 @RestAccount.action_registry.register('update')
 class UpdateAccount(BaseAction):
+    """Update the cloudwatch role associated to a rest account
+
+    :example:
+
+    .. code-block:: yaml
+
+        policies:
+          - name: reduce-account-burst-limie
```
Comment (kapilt) [inline c7n/resources/apigw.py]: policy name does not related to policy content for the example also s/limie/limit

File: c7n/resources/apigw.py (hunk 1 above)
Comment (joshuaroot) [reply c7n/resources/apigw.py]: Yeah, I think I can start with `- op:` and continue the dict below... i just got used to writing lists of dict in yaml like that.

**Comment by joshuaroot** [comment]
All of the examples have all been vetted and function correctly. I'll go back through and correct the policy names & look into the formatting...

**Comment by joshuaroot** [comment]
tested/validated the alternate method used here works exactly like previous for the `patch` values

**Pull request merged**